#!/usr/bin/env python
PACKAGE = "turtle_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Kv", double_t, 0, "Forward Gain",    1.35, 0,   5)
gen.add("Kh", double_t, 0, "Heading Gain",    0.75, 0,   5)
gen.add("x_vel_max", double_t, 0, "Linear Velocity Limit",    0.16, 0,   1)
gen.add("w_vel_max", double_t, 0, "Angular Velocity Limit",    4, 0.5,   5)

exit(gen.generate(PACKAGE, "turtle_controller", "Go2Goal"))